config/rbac/role.yaml

```diff
@@ -42,11 +42,19 @@ rules:
   resources:
   - components
   verbs:
+  - create
+  - delete
   - get
   - list
   - patch
   - update
   - watch
+- apiGroups:
+  - appstudio.redhat.com
+  resources:
+  - components/finalizers
```

Comment:
What does the slash mean here?  Does it means "This applies to components and finalizers for components?"

Reply:
It means finalizers for components, as that's a subresource.